README.md

```diff
@@ -174,45 +174,45 @@ Support for OpenGL accelerated rendering is made possible through the work our D
 
-At this point in time we have the following project mirrors for currently in-flight contributions.
+At this point in time, we have the following project mirrors for currently in-flight contributions.
 
 | Project | Upstream Repo | WSLg Mirror |
 |---|---|---|
 | Weston | https://github.com/wayland-project/weston | https://github.com/microsoft/Weston-mirror|
 | FreeRDP | https://github.com/FreeRDP/FreeRDP | https://github.com/microsoft/FreeRDP-mirror |
 | PulseAudio | https://github.com/pulseaudio/pulseaudio | https://github.com/microsoft/PulseAudio-mirror |
 
-Here's a high level overview of the currently in-flight contributions to each project contained within these mirrors.
+Here is a high level overview of the currently in-flight contributions to each project contained within these mirrors:
 
 ## Weston
-WSLg leverages Weston as the Wayland compositor bridging the Linux and Windows world using RDP technology to remote application content between them. Weston already had an RDP backend, but it was limited to single monitor desktop remoting. We've greatly enchanced that RDP backend to include advanced functionality, such as multi-monitor support, clipboard integration for cut/paste, audio in/out and enable a new remoting mode called RAIL (Remote Application Integrated Locally) and VAIL (Virtualized Application Integrated Locally) where individual applications, rather than desktop/monitor, are remoted. These changes are not really specific to WSLg, they essentially add functionality to the existing RDP backend and are reusable in other scenarios as well, like say using the new Weston RDP backend to remote application running on a Pi device to another device running an RDP client. 
+WSLg leverages Weston as the Wayland compositor bridging the Linux and Windows worlds using RDP technology to remote application content between them. Weston already had an RDP backend, but it was limited to single-monitor-desktop remoting. We've greatly enchanced that RDP backend to include advanced functionality, such as multi-monitor support, clipboard integration for copy/paste, and audio in/out. We've enabled new remoting modes called RAIL (Remote Application Integrated Locally) and VAIL (Virtualized Application Integrated Locally), where individual applications, rather than desktops/monitors, are remoted. These changes are not specific to WSLg; they add functionality to the existing RDP backend and are reusable in other scenarios as well (i.e. using the new Weston RDP backend to remote application running on a Raspberry Pi to another device running an RDP client).

```

Comment:
Typo: `enchanced` → `enhanced`
```suggestion
WSLg leverages Weston as the Wayland compositor bridging the Linux and Windows worlds using RDP technology to remote application content between them. Weston already had an RDP backend, but it was limited to single-monitor-desktop remoting. We've greatly enhanced that RDP backend to include advanced functionality, such as multi-monitor support, clipboard integration for copy/paste, and audio in/out. We've enabled new remoting modes called RAIL (Remote Application Integrated Locally) and VAIL (Virtualized Application Integrated Locally), where individual applications, rather than desktops/monitors, are remoted. These changes are not specific to WSLg; they add functionality to the existing RDP backend and are reusable in other scenarios as well (i.e. using the new Weston RDP backend to remote application running on a Raspberry Pi to another device running an RDP client).

```